src/pages/Product/ProductLists/ProductLists.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import ProductList from "./ProductList/ProductList";
+import "./ProductLists.scss";
+
+export default class ProductLists extends Component {
+  state = {
+    items: {},
+    filter: "",
+    view: 8,
+  };
+
+  handleClick = e => {
+    const clickedName = e.target.className;
+    this.setState({ ...this.state, filter: clickedName });
```

Comment:
저번에도 리뷰 드렸던 내용인데, 아래처럼만 사용하셔도 됩니다!

```suggestion
    this.setState({ filter: clickedName });
```

Reply:
아이고 넵넵!ㅠㅠ의식해서 변경하겠습니다!ㅠㅠㅠ 동 건 변경 완료입니다!